examples/ensemble/plot_ensemble_oob.py

```diff
@@ -81,7 +81,7 @@
 
 # Range of `n_estimators` values to explore.
 min_estimators = 15
-max_estimators = 175
+max_estimators = 125
 
 for label, clf in ensemble_clfs:
     for i in range(min_estimators, max_estimators + 1):
```

Comment:
We might not need to increase only 1 tree at a time but maybe 5. This should speed up the process since we will not call as many time the Python for a loop even if we have to train the same number of trees.